res/xml/urdu_phonetic_ur.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Urdu Phonetic" script="urdu">
+  <row>
+    <key key0="ق" key3=" " key4="esc" key7="1"/>
+    <key key0="و" key2="ؤ" key3="\@" key7="2"/>
```

Comment:
You've added more ASCII punctuations but not all. Some are missing: `` <, >, ?, `, |, ~ ``

Though, they're not mandatory if you wouldn't use them when writing Urdu.